src/lib/components/AnnouncementBanner/AnnouncementBanner.scss

```diff
@@ -0,0 +1,35 @@
+.usa-banner.ldaf-banner--announcement {
+  background: $primary-darkest;
+
+  .usa-banner__header-text,
+  .usa-banner__guidance {
+    color: white;
+  }
+
+  .usa-banner__button-text {
+    color: $primary-lighter;
+  }
+
+  .ldaf-logo {
+    filter: invert(1);
+  }
```

Comment:
What is this doing? It doesn't seem to be inverting the logo colors for me, if that's the intention.

Reply:
this is another case of the Icon component not supporting recoloring (see [LDAF-327](https://ldaf.atlassian.net/browse/LDAF-327)). Since this is just going from black to white, it's simpler than the `filter` we needed to use for the copy button. I'll add a TODO, as well

Reply:
Ah I gotcha. Thanks for adding those notes!